mets_mods2tei/api/mets.py

```diff
@@ -513,11 +516,17 @@ def get_encoding_description(self):
         """
         return self.encoding_desc
 
-    def get_owner_manuscript(self):
+    def get_location_phys(self):
         """
-        Return the owner of the original manuscript
+        Return the physical location of the original manuscript
```

Comment:
This looks like a complete change of the originally intended semantics of this function...

Reply:
How so?

Your name `owner` seemed arbitrary – in MODS it's `physicalLocation` only, in TEI it's [repository](https://www.deutschestextarchiv.de/doku/basisformat/mdSdMsDesc.html?hl=repository).

This change extends it to MODS' `url` for TEI's `idno`.